### ISSUE TITLE
man pages claim copyright for future years

### ISSUE THREAD

**Issue by bmwiedemann** [ClusterLabs/libqb#414]
While working on the reproducible builds effort, I found that
when building the libbq package for openSUSE Linux, there were differences between builds:

```diff
+++ new//usr/share/man/man3/qbutil.h.3  2020-09-03 20:00:09.988392060 +0000
@@ -17,4 +17,4 @@
 .hy
 .SH "COPYRIGHT"
 .PP
-Copyright (C) 2010-2020 Red Hat, Inc. All rights reserved.
+Copyright (C) 2010-2035 Red Hat, Inc. All rights reserved.
```

Code should not claim copyright for future years.

**Comment by chrissie-c** [comment]
That's interesting!

The man pages are generated using doxygen2man and the 'to' date is usually taken from the current date so it should always be correct if the computer date is set right, so check that first. If doxygen2man is picking up an wrong date from somwhere though I'd be interested to know how.

**Comment by chrissie-c** [comment]
I've created a PR that might help this https://github.com/ClusterLabs/libqb/pull/415, and testing you can give would be much appreciated :)

**Comment by chrissie-c** [comment]
Fixed in PR#415, thanks for help with testing.
